sw/device/examples/sram_program/BUILD

```diff
@@ -188,7 +188,7 @@ OTP_IMAGES = [
 test_suite(
     name = "rom_e2e_jtag_inject_tests",
     tags = [
```

Comment:
test suite tags are confusing and filter the contents of the test suite AFAIK

Reply:
Oof, I did not realize that. Thanks for the heads up.

Reply:
I figure there's no value in specifying `tags` here then, so I'll delete it.

Reply:
Thanks for the heads up @drewmacrae!